configs/RADIOLINKF722/config.h

```diff
@@ -0,0 +1,116 @@
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F7X2
+
+#define BOARD_NAME        RADIOLINKF722
+#define MANUFACTURER_ID   RALI
+
+#define USE_GYRO
+#define USE_ACC
+#define USE_ACCGYRO_BMI270
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC_SPI_ICM42688P
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
+#define USE_FLASH
+#define USE_FLASH_W25N01G
+#define USE_MAX7456
+
+#define BEEPER_PIN           PC13
+#define MOTOR1_PIN           PA15
+#define MOTOR2_PIN           PB3
+#define MOTOR3_PIN           PB4
+#define MOTOR4_PIN           PB6
+#define MOTOR5_PIN           PB7
+#define MOTOR6_PIN           PB5
+#define MOTOR7_PIN           PB0
+#define MOTOR8_PIN           PB1
+#define RX_PPM_PIN           PA3
+#define LED_STRIP_PIN        PC8
+#define UART1_TX_PIN         PA9
+#define UART2_TX_PIN         PA2
+#define UART3_TX_PIN         PB10
+#define UART6_TX_PIN         PC6
+#define UART1_RX_PIN         PA10
+#define UART2_RX_PIN         PA3
+#define UART3_RX_PIN         PB11
+#define UART4_RX_PIN         PA1
+#define UART6_RX_PIN         PC7
+#define I2C1_SCL_PIN         PB8
+#define I2C1_SDA_PIN         PB9
+#define LED0_PIN             PA14
+#define SPI1_SCK_PIN         PA5
+#define SPI2_SCK_PIN         PB13
+#define SPI3_SCK_PIN         PC10
+#define SPI1_SDI_PIN         PA6
+#define SPI2_SDI_PIN         PB14
+#define SPI3_SDI_PIN         PC11
+#define SPI1_SDO_PIN         PA7
+#define SPI2_SDO_PIN         PB15
+#define SPI3_SDO_PIN         PC12
+#define CAMERA_CONTROL_PIN   PA0
+#define ADC_VBAT_PIN         PC2
+#define ADC_RSSI_PIN         PC0
+#define ADC_CURR_PIN         PC1
+#define SDCARD_SPI_CS_PIN    PD2
+#define PINIO1_PIN           PC9
+#define FLASH_CS_PIN         PD2
+#define MAX7456_SPI_CS_PIN   PB12
+#define GYRO_1_EXTI_PIN      PC4
+#define GYRO_1_CS_PIN        PB2
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP( 0, PA15, 1,  0) \
+    TIMER_PIN_MAP( 1, PB3 , 1,  0) \
+    TIMER_PIN_MAP( 2, PB4 , 1,  0) \
+    TIMER_PIN_MAP( 3, PB6 , 1,  0) \
+    TIMER_PIN_MAP( 4, PB7 , 1,  0) \
+    TIMER_PIN_MAP( 5, PB5 , 1,  0) \
+    TIMER_PIN_MAP( 6, PB0 , 2,  0) \
+    TIMER_PIN_MAP( 7, PB1 , 2,  0) \
+    TIMER_PIN_MAP( 8, PC8 , 2,  0) \
+    TIMER_PIN_MAP( 9, PA3 , 3, -1) \
+    TIMER_PIN_MAP(10, PA0 , 2,  0)
```

Comment:
This does not need DMA: ` TIMER_PIN_MAP(10, PA0 , 2,  -1)`

Reply:
Do not delete it, just change the line as shown so that it does not request DMA for this timer.

Reply:
Modified to TIMER_PIN_MAP(9, PA0 , 2,  -1)